### PR TITLE
First pass of loading spinner component.

### PR DESCRIPTION
**Pull request by christianmemije**
## Summary

* Adds loading spinner component. 
* Can be used throughout Kolibri using `require('loading-spinner')`
* Used while content loads and while video player loads.
* Temporarily removed video poster.

## TODO

- [ ] Styling

## Reviewer guidance

* Are there files that I touched that I should have not?
* Best use of `v-if` and `v-show`

## Issues addressed

* https://trello.com/c/FlkXs3E0/513-re-usable-scalable-spinner-component-for-placeholding-while-data-content-is-being-loaded

### While content loads.
image

### While video player loads.
image

### CODE REVIEW

**Comment by rtibbles** [comment]
@christianmemije Is this based off latest master?

I am getting a lot of console warnings when I run your branch:

image

**Comment by rtibbles** [comment]
Double checked on master, and didn't see any.

**Comment by christianmemije** [comment]
Pushed a change a few seconds ago to fix that ^

**Comment by rtibbles** [comment]
Yep, definitely did!

File: frontend_build/src/read_bundle_plugins.js [hunk 1]
```diff
@@ -27,6 +27,7 @@ var libs = function(kolibri_name) {
     'vuex': kolibri_name + '.lib.vuex',
     'core-base': kolibri_name + '.lib.coreBase',
```
Comment (rtibbles) [inline frontend_build/src/read_bundle_plugins.js]: Bah, I missed deleting these three lines. Could you remove them, confirm that the build still works (after a restart) and push again?

Comment (rtibbles) [inline frontend_build/src/read_bundle_plugins.js]: 28-30

**Comment by christianmemije** [comment]
@rtibbles Committed, still works.

**Comment by rtibbles** [comment]
To address @indirectlylit's comment here: https://trello.com/c/FlkXs3E0/513-re-usable-scalable-spinner-component-for-placeholding-while-data-content-is-being-loaded#comment-57bb854923eac0d5473ba600

Perhaps you could also add the loading gif in the base.html inside the `<vueroot>` component (assuming that once the router mounts there, it will wipe anything inside).

**Comment by christianmemije** [comment]
@rtibbles Added the spinner to base.html

**Comment by rtibbles** [comment]
Works!

**Pull request merged**